ci/test/00_setup_env_native_tidy.sh

```diff
@@ -15,5 +15,5 @@ export RUN_FUNCTIONAL_TESTS=false
 export RUN_FUZZ_TESTS=false
 export RUN_TIDY=true
 export GOAL="install"
-export BITCOIN_CONFIG="CC=clang CXX=clang++ --with-incompatible-bdb --disable-hardening CFLAGS='-O0 -g0' CXXFLAGS='-O0 -g0'"
+export BITCOIN_CONFIG="CC=clang CXX=clang++ --enable-c++20 --with-incompatible-bdb --disable-hardening CFLAGS='-O0 -g0' CXXFLAGS='-O0 -g0'"
```

Comment:
@0xB10C @MarcoFalke 

What was the point to add `--enable-c++20` here?

Asking in context of https://github.com/bitcoin/bitcoin/pull/26763#discussion_r1058860943.

Reply:
It wasn't added in this pull, see the diff in `ci/test/00_setup_env_native_asan.sh`.

It was added to check that compiling with it works.

Reply:
Could another CI task use it instead of the "tidy" one? (fwiw, we have a native Windows build using C++20)

Reply:
Sure, any Linux task that builds all targets (including fuzz and gui) should do. For example, multiprocess, asan, or tsan.

Reply:
> It wasn't added in this pull, see the diff in `ci/test/00_setup_env_native_asan.sh`.

Suggesting to revert the move of `--enable-c++20` between CI tasks in #26770.